Options.lua

```diff
@@ -9,32 +9,134 @@ local L = LibStub("AceLocale-3.0"):GetLocale("TeleportMenu")
 -------------------------------------
 -- Locales
 --------------------------------------
-
-local defaultsDB = {
-    enabled = true,
-    iconSize = 40,
-    hearthstone = "none",
-    maxFlyoutIcons = 5,
-    reverseMageFlyouts = false,
-    buttonText = true,
-    showOnlySeasonalHerosPath = false
+SettingsBase = {
+	__index = {
+		["Enabled"] = false,
+		["Teleports:Seasonal:Only"] = false,
+		["Teleports:Mage:Reverse"] = false,
+		["Teleports:Hearthstone"] = "none",
+		["Button:Size"] = 40,
+		["Button:Text:Size"] = 14,
+		["Button:Text:Show"] = true,
+		["Flyout:Max_Per_Row"] = 5,
+        [202046] = false,     -- Lucky Tortollan Charm
```

Comment:
different indentation again, however I think this itemlist should be moved to the items.lua file - it's messy in here and will be added/updated more frequent than normal settings.